data_prep_tools.py

```diff
@@ -1,31 +1,78 @@
-from nfldb_tables import NFLFrames
 import pandas as pd
+from factorize import nmf_all_positions
 
-positions_list = ['QB', 'TE', 'RB', 'K', 'WR']
 
 def get_yr_until_wk(year, till_week, nfl_frames):
     '''
-    Input: Integers denoting the year/season and what week to collect data to.
-            Optionally NFLFrames instance (to save cost of building new one)
-    Output:  DataFrame containing player_ids, opponent and fanduel points for season 
-    up until week in question
+    Input:  Int, Int - week to collect data to, NFLFrames
```

Comment:
This was actually intentional. If you have it as a default value, when the function is brought into the namespace it runs the nfl_frames constructor whether or not you end up needing it...